internal/controllers/utils/constants.go

```diff
@@ -124,3 +124,9 @@ const (
 	ClusterInstanceDataType = "ClusterInstance"
 	PolicyTemplateDataType  = "PolicyTemplate"
 )
+
+// Environment variable names
+const (
+	TLSSkipVerifyEnvName      = "INSECURE_SKIP_VERIFY"
+	TLSSkipVerifyDefaultValue = true // TODO(alegacy): need to set to false for production
```

Comment:
I'm going to disable this, but first I need to investigate why we are failing to validate the cert.